### PR TITLE
tests: load and install dependencies for docker tests

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: .github/workflows/docker-test.yml [hunk 1]
```diff
@@ -14,15 +14,30 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Start containers
+      - name: Install requirements
+        # make sure nvme-cli installed (we need it for /etc/nvme/hostnqn and /etc/nvme/hostid)
```
Comment (martin-belanger) [inline .github/workflows/docker-test.yml:18]: Actually, you can use `stasadm` to create /etc/nvme/[hostnqn|hostid]`

File: .github/workflows/docker-test.yml [hunk 2]
```diff
@@ -14,15 +14,30 @@ jobs:
...
+        run: sudo apt-get install -y nvme-cli
+
+      - name: Load Kernel drivers
+        run: sudo modprobe -v nvme-fabrics
+
+      - name: Fix D-BUS
+        run: |
+          sed 's/@STAFD_DBUS_NAME@/org.nvmexpress.staf/g' etc/dbus-1/system.d/org.nvmexpress.staf.in.conf | sudo tee /usr/share/dbus-1/system.d/org.nvmexpress.staf.conf
```
Comment (martin-belanger) [inline .github/workflows/docker-test.yml:26]: I'm not sure why this is needed. The installed D-Bus conf files should not have any parameters like `@STAFD_DBUS_NAME@` in them?

File: .github/workflows/docker-test.yml (hunk 1 above)
Comment (glimchb) [reply .github/workflows/docker-test.yml:18]: yep, saw it. we will need nvme-cli anyway for later tests, so decided to install it

File: .github/workflows/docker-test.yml (hunk 2 above)
Comment (glimchb) [reply .github/workflows/docker-test.yml:26]: the files installed inside the container but we need them on the host... so either this or copy from inside container to the host... this is better i think

**Pull request merged**